lib/backends/sdc/utils.js

```diff
@@ -29,6 +29,34 @@ function networkUuidToDockerId(uuid) {
     return (uuid + uuid).replace(/-/g, '');
 }
 
+function shortNetworkIdToUuid(id) {
+    var partial = '';
+
+    // consider a uuid as eight groups of four chars, plus hyphens.
+    // concatenate those groups in reverse order, jumping to the
+    // appropriate starting group.
+
+    switch (Math.ceil(id.substr(0, 32).length / 4)) {
```

Comment:
Could this be achieved by using a regular expression similar to:

````
var matches = id.match(/([a-z0-9]{0,8})?([a-z0-9]{0,4})?.../);
````

and then iterate through from `matches[1]` through `matches.length` to reconstruct a UUID?

I feel like it could be easier to understand, but I realize this is subjective.